doc/specs/SCOOBE.md

```diff
@@ -0,0 +1,185 @@
+# PowerToys SCOOBE Dialog
+
+- **What is it:**  Post-upgrade prompted dialog that walks users through the latest additions to PowerToys utilities and functionalities
+- **Author:**  Deondre Davis
+- **Spec Status:**  Draft
+
+## 1. Overview
+
+### 1.1. Executive Summary
+
+To expand on our end-user onboarding efforts, we seek to resolve the critical issue of informing users of new additions and improvements to PowerToys during upgrade scenarios. We currently list release notes on the main repository, in addition to affiliated information on Microsoft Docs. However, neither of these mediums are internal to the application, and thus requires users to proactively seek out information on what&#39;s been updated, or merely notice by chance what&#39;s been changed from their regular usage. As these are not ideal experiences, this document describes the inclusion of a post-upgrade &#39;SCOOBE&#39; prompt that launches the previously developed OOBE dialog loaded with new information related to upgrade improvements and additions.
+
+### 1.2. Key-Definitions/Concepts
+
+Here are definitions for words and acronyms found throughout this document to ensure clarity:
+
+- **OOBE:** Out of box experience – The users&#39; initial interactions with the product immediately after installing the product and/or launching the product for the first time.
+- **SCOOBE:** Second chance out of box experience - The users&#39; initial interactions with the product immediately after upgrading the product and/or launching the product for the first time after upgrading.
+
+### 1.3. Goals and Non-Goals
+
+Goals:
+
+- Create a guided prompt that exposes the user to a brief overview of the new features and/or improvements included with the latest version of PowerToys.
+
+Non-Goals:
+
+- Present a copy-and-paste replica of the repository release notes. This information needs to be readily consumable, often requiring visual demonstrations of the new behavior and/or functionality that&#39;s either not possible or not necessary to depict on the repository release notes.
```

Comment:
I think that can be achieved by adding images of the new features and links like "try it out"